pages/playback/controls/player/popup-input/basic-input.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <div>
+    <PopInput
+      :title_label="label"
+      :key_placeholder="keyplaceholder"
+      :input_check="spellchecking"
+      :invalid_text="error_text"
+      :user_key.sync="entrykey"
+      :input_width="entry_width"
+      :block="disallow_entry"
+    ></PopInput>
+  </div>
+</template>
+
+<script>
+import { PopInput } from "../../../../../dist/mantarray.common";
```

Comment:
you can use `from "@/dist/mantarray.common"` similar to `@/components` .  That way if this file gets moved to reorganize the folder structure, it will still correctly point to the `dist` folder without having to modify the file path